src/WeatherApi/WeatherApi.csproj

```diff
@@ -11,8 +11,6 @@
   <!-- Add support for Ahead-of-Time (AoT) compilation -->
   <PropertyGroup Condition="!$([System.OperatingSystem]::IsMacOS())">
     <InvariantGlobalization>true</InvariantGlobalization>
-    <!-- HACK Suppress IL warnings from Microsoft.Extensions.Http.Resilience until 8.0.0 -->
```

Comment:
If you want to disable warnings coming from a specific set of libraries, a strategy I found that works is to use `<TrimmerSingleWarn>false</TrimmerSingleWarn>` and disable the warning for `IL2104` (the warning number for a single warning for a whole library).

And then turn on "single warnings" for the specific libraries you want to suppress. You can see an example of that here:
https://github.com/dotnet/eShop/blob/1e803b09ba8caa3702f037deac837fab956abe33/Directory.Build.targets#L7-L23